### DISCUSSION TITLE
fatal error: ADSingleVariableReturnMappingSolution.h: No such file or directory

### DISCUSSION THREAD

**Discussion by zxj19910214** [idaholab/moose#27986] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (avoid screenshots if possible, triple back quotes before/after pasted text, etc)

### Issue or question about MOOSE

Dear mooseusers,

when I try to compile files from others, it shows this error. 

could you guide me to fix this?

Best regards,
XIao

### Diagnostics for MOOSE installation issues

[Optional] Output of the diagnostics scripts. To run the script: cd ~/projects/moose/scripts; ./diagnostics.sh

**Comment by GiudGiud** [comment]
Hello

Can you please paste the full compilation log?
Is thris your app or MOOSE or a module?

**Reply by zxj19910214** [reply, marked as answer]
Dear,

Thank you for helping me with this.  I found it is because the module tensor mechanics is not included when I compile the app.

Have a good day!
Xiao
